### DISCUSSION TITLE
Windows tasbkbar's window preview shows all tabs from all workspaces

### DISCUSSION THREAD

**Discussion by ahmaadaziz** [zen-browser/desktop#2794] [Ideas]
### What happened?

Windows' taskbar window preview shows all tabs from all workspaces, making it difficult to navigate to the correct one.

**Expected Behaviour:**
Only the tabs from the active workspace should show in the preview  

### Reproducible?

- [X] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.0-a.28

### Severity impact

Medium

### What platform are you seeing the problem on?

Windows

**Comment by salamijason** [comment]
Heylo, looks like this post was made a while back. Are you still getting this issue? (latest ver a.17 as of this comment)

I previously helped someone else who had this issue:

1) make sure this option is unchecked in the settings: ![Image](https://github.com/user-attachments/assets/231861ee-3146-4366-8b79-8946d0735fd1)

2) if the above fix doesn't do the trick, try a new profile: https://docs.zen-browser.app/guides/manage-profiles

**Comment by ahmaadaziz** [comment]
Yes, that did work, now it just shows the active tab which is to be expected when "Show tab previews in the Windows taskbar" is disabled. 
However, the underlying issue still exists when the option is enabled; basically, I want tab previews for my active workspace to be shown. For now, (when enabled) it shows two additional previews that just have a Windows' loading spinner on them (two more because I have two more workspaces).

But this will do for now I guess, thanks for highlighting the fix 🙌
Do you want me to close the issue? I think it should stay open because it is in the public roadmap as well now.
@mauro-balades The issue is present under more information is needed, can you tell me what you need. I'd be happy to help

**Comment by salamijason** [comment]
Ohh I see I see. I think that'd be a cool addition. Unfortunately, I will have to close this issue 🙏  I'll move this to discussions where you can elaborate on what your desired behavior for the browser is. 
We're having to go through hundreds of issues right now so I can't promise when this will receive attention. Cheff also has about 6 dozen bugs to fix and go through 😅 

But thank you for the post!!

**Comment by barkely128** [comment]
Guys this would not be a **cool addiiton**, it is, in fact, a **regression** from Firefox. The vanilla browser is able to show only active window's tabs and Zen isn't.

I absolutely want to divide windows, along with their tabs, depending on their context. Like come on.
